PerDiem/PerDiem/TransactionCell.m

```diff
@@ -14,6 +25,27 @@ - (void)awakeFromNib {
     // Initialization code
 }
 
+- (void)setTransaction:(Transaction *)transaction {
+    _transaction = transaction;
+
+    self.dateLabel.text = [self formatDate:self.transaction.transactionDate];
+    self.budgetLabel.text = self.transaction.budget.name;
+    self.paymentTypeLabel.text = self.transaction.paymentType.name;
+    self.summaryLabel.text = self.transaction.summary;
+
+    // @TODO - make common currency formatter method to be used throughout app.
```

Comment:
We should make some common helpers for currency and date formatting. 
Also, we should figure out the best way to easily make all currency strings localized.